routes/SessionRoute.py

```diff
@@ -34,14 +33,14 @@
             "properties": {
                 "startCapital": {
                     "type": "integer",
-                    "min": 0,
+                    "minimum": 0,

```

Comment:
`min` does not work, it is `minumum`

Reply:
closes https://github.com/Python-Project-Thomas-More-year-2/backend-flask-Python-Project-Thomas-More-year-2/issues/28